### PR TITLE
Added functionality for TV Episodes

### PR DESCRIPTION
**Pull request by ttufts**
Added get_episodes to imdbpie, added Episode object, added unit tests for new function and new object, added test data for episodes

### CODE REVIEW

File: src/imdbpie/imdbpie.py [hunk 1]
```diff
@@ -35,7 +35,7 @@ def __init__(self, api_key=None, locale=None, anonymize=None,
         self.caching_enabled = True if cache is True else False
         self.proxy_uri = proxy_uri or DEFAULT_PROXY_URI
         self.anonymize = False or anonymize
-        self.verify_ssl = True or verify_ssl
+        self.verify_ssl = False if verify_ssl is not None else True
```
Comment (richardasaurus) [inline src/imdbpie/imdbpie.py]: A nicer way would be to just give the param a default `verify_ssl=True`, then this line would be:

```python
self.verify_ssl = verify_ssl
```

File: src/imdbpie/imdbpie.py [hunk 2]
```diff
@@ -209,6 +209,32 @@ def get_person_images(self, imdb_id):
         response = self._get(url)
         return self._get_images(response)
 
+    def get_episodes(self, imdb_id):
+        title = self.get_title_by_id(imdb_id)
+        if title.type != "tv_series":
+            print("Title provided is not of type TV Series")
+            return None
```
Comment (richardasaurus) [inline src/imdbpie/imdbpie.py]: can you make this raise a `RuntimeError('Title provided is not of type TV Series')` instead? as if you're trying to get an episode for a non series type Title then your code is wrong

**Comment by richardasaurus** [comment]
Looks good except those small things. Also some conficts with master

**Comment by ttufts** [comment]
Recommended changes made. Can you handle the conflicts with master?

**Comment by richardasaurus** [comment]
Github won't let me click merge until they're resolved

**Comment by ttufts** [comment]
Github won't let me merge it because I don't have write access to master.

**Comment by richardasaurus** [comment]
@ttufts you need to just merge `master` into your branch, resolve conflicts, push

**Comment by ttufts** [comment]
Got it. All of those were in tests that I had fixed, but apparently you fixed them too, just in a different way. I went with yours on all of them.

**Pull request merged**